seleniumJavaExperiment/.gitignore

```diff
@@ -0,0 +1,169 @@
+
+# Created by https://www.toptal.com/developers/gitignore/api/intellij,macos,java
+# Edit at https://www.toptal.com/developers/gitignore?templates=intellij,macos,java
+
+### Intellij ###
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio, WebStorm and Rider
+# Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
```

Comment:
@isramodus could you remove this please

Reply:
I believe I have solved the issues @pedrohyvo . Could you please check again? Thanks! 🙏